pkg/cmd/roachtest/monitor.go

```diff
@@ -163,6 +163,17 @@ func (m *monitorImpl) ExpectProcessAlive(nodes option.NodeListOption, opts ...op
 	m.ExpectProcessHealth(nodes.InstallNodes(), ExpectedAlive, opts...)
 }
 
+func (m *monitorImpl) AvailableNodes(virtualClusterName string) option.NodeListOption {
```

Comment:
Nice!

nit: leave a comment explaining what availableNodes does, specifically mentioning how it's availability based on the expected health, not based on what the monitor actually observes.